arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -23,178 +23,227 @@
           }
         },
-        "deprecated": false,
-        "x-ms-long-running-operation": true
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create failover group": {
+            "$ref": "../examples/FailoverGroupCreateOrUpdate.json"
+          }
+        }
       },
       "delete": {
         "tags": [
           "FailoverGroups"
         ],
         "description": "Deletes a failover group.",
         "operationId": "FailoverGroups_Delete",
-        "x-ms-examples": {
-          "Delete failover group": {
-            "$ref": "../examples/FailoverGroupDelete.json"
-          }
-        },
         "parameters": [
           {
-            "$ref": "#/parameters/ApiVersionParameter"
```

Comment:
nit: maintain a consistent parameter ordering, either place ApiVersion and SubscriptionId together at the beginning of the parameters section or at the end.